pkg/audit/manager.go

```diff
@@ -432,7 +432,7 @@ func (ucloop *updateConstraintLoop) updateConstraintStatus(ctx context.Context,
 	var statusViolations []interface{}
 	for _, ar := range auditResults {
 		// append statusViolations for this constraint until constraintViolationsLimit has reached
-		if len(statusViolations) < *constraintViolationsLimit {
+		if len(statusViolations) < int(*constraintViolationsLimit) {
```

Comment:
We should leave this as uint to avoid integer overflow issues (not that anyone should express a limit that high)

Reply:
ah, I see the type mismatch issue... if we cast len(statusViolations) to uint, we avoid overflow